saos-webapp/src/main/webapp/WEB-INF/view/template/skipLinks/defaultSkipLinks.jsp

```diff
@@ -0,0 +1,22 @@
+<%@ page language="java" contentType="text/html; charset=UTF-8" pageEncoding="UTF-8"%>
```

Comment:
What is the reasoning behind defaultSkipLinks? Shouldn't each page have it own special skipLinks? I guess defaultSkipLinks gives us nothing and even hides the need of providing custom skipLinks for each page separately.